api/v1beta2/ocicluster_types.go

```diff
@@ -120,6 +126,46 @@ type OCIAvailabilityDomain struct {
 	FaultDomains []string `json:"faultDomains,omitempty"`
 }
 
+// ClusterClientHostUrls contains information about client host url overrides.
+type ClusterClientHostUrls struct {
```

Comment:
I dont think we need Cluster  here, can we call this ClientURLMap, starting a discussion here so that we can brainstorm and get to better name, we should remove cluster anyways as we have to level Cluster.

Reply:
What about `ClientHostOverrides` Then we might have:
```
ClientHostOverrides:
    ComputeClientUrl:
````

I don't love the "overrides" but it is more clear what this map is used for 🤷‍♂️

Reply:
I like ClientOverrides

Reply:
`ClientOverrides` is nice in that we can expand past URLs if needed 🤷 
certs, ports, etc